timvt/endpoints/tiles.py

```diff
@@ -30,8 +30,11 @@ async def tile(
     db_pool: Pool = Depends(_get_db_pool),
 ) -> TileResponse:
     """Handle /tiles requests."""
-    if not re.match(r"^[a-z]+[a-z_\-0-9]*$", table, re.I):
-        raise Exception("Bad tablename")
+
+    idx = await index(db_pool)
+    if table not in idx:
+        raise Exception("Table not found")
```

Comment:
@bitner instead of listing all tables can't we just check our table exist with a simple SQL query?